core/ecschema-editing/package.json

```diff
@@ -35,7 +35,7 @@
     "url": "http://www.bentley.com"
   },
   "devDependencies": {
-    "@bentley/units-schema": "^1.0.7",
+    "@bentley/units-schema": "^1.0.8",
```

Comment:
`rush install` might also fail in the build. You can check it locally and may have to do `rush update` to fix it.

Reply:
The devDependencies are working fine as we've published the `Units v1.0.8` package. Also, I've updated the change description using `rush change`.

Reply:
yes, I can see now `pnpm-lock.yaml` has updated so the build should be fine.